scripts/fill_empty_postgresql_tables.py

```
@@ -46,9 +63,9 @@ def get_empty_tables(self, db):
 
         empty_tables = []
         for row in result:
-            table_name = row[0]
-            count_result = db.fetch_one(f"SELECT COUNT(*) FROM {table_name}")
-            if count_result and count_result[0] == 0:
+            table_name = _scalar(row, "table_name")
+            count = _scalar(db.fetch_one(f"SELECT COUNT(*) FROM {table_name}")) or 0
```

Ruff (0.15.12)

[error] Possible SQL injection vector through string-based query construction

(S608)